sympy/functions/special/delta_functions.py

```diff
@@ -377,24 +393,32 @@ def _sage_(self):
 
 
 class Heaviside(Function):
-    """Heaviside Piecewise function
+    """
+    Heaviside Piecewise function.
 
-    Heaviside function has the following properties [1]_:
+    Explanation
+    ===========
+
+    Heaviside function has the following properties:
 
     1) ``diff(Heaviside(x),x) = DiracDelta(x)``
                         ``( 0, if x < 0``
     2) ``Heaviside(x) = < ( undefined if x==0 [1]``
                         ``( 1, if x > 0``
     3) ``Max(0,x).diff(x) = Heaviside(x)``
```

Comment:
Let's go ahead and fix this as well. This is supposed to be showing math, but it renders really bad and is very difficult to read. Here is the equivalent LaTeX for each of these

1. \frac{d}{d x} \theta(x) = \delta(x)
2. \theta(x) = \begin{cases} 0 & \text{for}\: x < 0 \\ \text{undefined} & \text{for}\: x = 0 \\1 & \text{for}\: x > 0 \end{cases}
(it looks like you've removed the footnote)
3. \frac{d}{d x} \max(x, 0) = \theta(x)

Also we should change `H(x)` to `\theta(x)` (in LaTeX math) below, since that is what the SymPy latex() printer uses. There should be a sentence before the first use explaining that Heaviside(x) is printed as \theta(x).